script.js

```diff
@@ -1,4 +1,495 @@
+  }
+  return endResult;
+};
+
+//determine round result win lose or tie
+var checkRoundResult = function (deckValue1, deckValue2) {
+  console.log("SEND HELP 3");
+  var playerValue = addAllCards(deckValue1);
+  var computerValue = addAllCards(deckValue2);
+
+  var roundResult;
+
+  /*
+  if (
+    (playerValue == 21 && computerValue != 21) ||
+    (playerValue < 21 && computerValue > 21) ||
+    playerValue > computerValue
+  ) {
+    roundResult = WIN;
+  } else if (
+    (playerValue != 21 && computerValue == 21) ||
+    (playerValue > 21 && computerValue < 21) ||
+    playerValue < computerValue
+  ) {
+    roundResult = LOSE;
+  } else if (playerValue == computerValue) {
+    roundResult = TIE;
+  }
+  
+  */
+  if (playerValue == 21 && computerValue != 21) {
+    roundResult = WIN;
+  } else if (playerValue != 21 && computerValue == 21) {
+    roundResult = LOSE;
+  } else if (playerValue > 21 && computerValue < 21) {
+    roundResult = LOSE;
+  } else if (playerValue < 21 && computerValue > 21) {
+    roundResult = WIN;
+  } else if (playerValue == 21 && computerValue == 21) {
+    roundResult = TIE;
+  } else if (playerValue == computerValue) {
+    roundResult = TIE;
+  } else if (playerValue > 21 && computerValue > 21) {
+    roundResult = TIE;
+  } else if (playerValue < computerValue) {
+    roundResult = LOSE;
+  } else if (playerValue > computerValue) {
+    roundResult = WIN;
+  }
+  return roundResult;
+};
+
+//determine what image for win lose or tie
+var checkImageResult = function (deckValue1, deckValue2) {
+  console.log("SEND HELP 4");
+  var playerValue = addAllCards(deckValue1);
+  var computerValue = addAllCards(deckValue2);
+
+  var imageResult;
+
+  /*
+  if (
+    (playerValue == 21 && computerValue != 21) ||
+    (playerValue < 21 && computerValue > 21) ||
+    playerValue > computerValue
+  ) {
+    imageResult = IMAGE_WIN;
+  } else if (
+    (playerValue != 21 && computerValue == 21) ||
+    (playerValue > 21 && computerValue < 21) ||
+    playerValue < computerValue
+  ) {
+    imageResult = IMAGE_LOSE;
+  } else if (playerValue == computerValue) {
+    imageResult = IMAGE_TIE;
+  }
+  return imageResult;
+  */
+  if (playerValue == 21 && computerValue != 21) {
+    imageResult = IMAGE_WIN;
+  } else if (playerValue != 21 && computerValue == 21) {
+    imageResult = IMAGE_LOSE;
+  } else if (playerValue > 21 && computerValue < 21) {
+    imageResult = IMAGE_LOSE;
+  } else if (playerValue < 21 && computerValue > 21) {
+    imageResult = IMAGE_WIN;
+  } else if (playerValue == 21 && computerValue == 21) {
+    imageResult = IMAGE_TIE;
+  } else if (playerValue == computerValue) {
+    imageResult = IMAGE_TIE;
+  } else if (playerValue > 21 && computerValue > 21) {
+    imageResult = IMAGE_TIE;
+  } else if (playerValue < computerValue) {
+    imageResult = IMAGE_LOSE;
+  } else if (playerValue > computerValue) {
+    imageResult = IMAGE_WIN;
+  }
+  return imageResult;
+};
```

Comment:
Most of the code is similar to `checkRoundResult()` - can we set the image in that function instead?